File: Source/server/rabbit-game/src/SharedModel/GameDetails.cs
namespace RabbitGameServer.SharedModel
{

	public class GameDetails
	{
		public string gameId { get; set; }

		public string roomName { get; set; }
		public string master { get; set; }
		public string winner { get; set; }
		public double msDuration { get; set; }
		public DateTime startDate { get; set; }
		public int pointsGain { get; set; }

		public GameDetails(string gameId,
			string roomName,
			string master,
			string winner,
			double duration,
			DateTime startDate,
			int pointsGain)
		{
			this.gameId = gameId;
			this.roomName = roomName;
			this.master = master;
			this.winner = winner;
			this.msDuration = duration;
			this.startDate = startDate;
			this.pointsGain = pointsGain;
		}
	}
}

File: Source/root/src/main/java/root/communication/messages/GameDetails.java
package root.communication.messages;

import java.util.Date;

public class GameDetails {

	public String gameId;

	public String roomName;
	public String master;
	public String winner;
	public double msDuration;
	public int pointsGain;
	public Date startDate;

	public GameDetails() {
	}

	public GameDetails(String gameId,
			String roomName,
			String master,
			String winner,
			double msDuration,
			int pointsGain,
			Date startDate) {

		this.gameId = gameId;
		this.roomName = roomName;
		this.master = master;
		this.winner = winner;
		this.msDuration = msDuration;
		this.pointsGain = pointsGain;
		this.startDate = startDate;
	}

}
